mahiru/components/settings.py

```diff
@@ -0,0 +1,126 @@
+"""Site configuration."""
+from pathlib import Path
+from typing import AnyStr, IO, List, Optional, Union
+
+import ruamel.yaml as yaml
+import yatiml
+
+from mahiru.definitions.identifier import Identifier
+
+
+class AssetConnections:
```

Comment:
```suggestion
class AssetConnectionSettings:
```
?? To make it clear this is settings?

Reply:
Or you could refer to it as `settings.AssetConnections`

Reply:
Yeah, I'm not very happy with the naming here either. Maybe `NetworkSettings`? That's shorter than `AssetConnectionSettings` but still covers the contents.